build-system/release-tagger/update-issue-tracker.js

```diff
@@ -102,6 +102,10 @@ class IssueTracker {
  * @return {Promise<void>}
  */
 async function createOrUpdateTracker(head, base, channel, time) {
+  const timeET =
+    new Date(`${time} UTC`).toLocaleString('en-US', {
+      timeZone: 'America/New_York',
+    }) + ' ET';
```

Comment:
All of the project's schedules are in PT 😢

```suggestion
  const timePT =
    new Date(`${time} UTC`).toLocaleString('en-US', {
      timeZone: 'America/Los_Angeles',
    }) + ' PT';
```

Reply:
Hokay fine